suites/quincy/nvmeof/ceph_nvmeof_deployment.yaml

```diff
@@ -99,3 +94,26 @@ tests:
       module: test_ceph_nvmeof_gateway.py
       name: configure Ceph NVMEoF Gateway
       polarion-id:
+
+  # Cleanup Test Case
+  - test:
+      abort-on-fail: true
+      config:
+        gw_node: node6
+        rbd_pool: rbd
+        do_not_create_image: true
+        rep-pool-only: true
+        rep_pool_config:
+        cleanup:
```

Comment:
Few comments here ,
Referring to http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-7KZGNZ/Test_to_cleanup_the_nvme-of_entities_0.log seems like there are 

- additional pool (rep_pool_bFSxgBoOLr)getting created and initialized in cleanup test which may not be expected here
- delete subsystem is deleting other subsystem(nqn.2016-06.io.spdk:cnode2) than the one specified in test  
- on client, nvme disconnect is getting applied on other subsystem(nqn.2016-06.io.spdk:cnode2) than one specified in test 
- additionally nvme-cli is getting installed again if I am not wrong.

Could you please check these if this is the case?

Reply:
> Few comments here , Referring to http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-7KZGNZ/Test_to_cleanup_the_nvme-of_entities_0.log seems like there are
> 
Inline my comments ---
> * additional pool (rep_pool_bFSxgBoOLr)getting created and initialized in cleanup test which may not be expected here------This could be avoided I agree. Need to check with Sunil on this module how hes calling this and created, I could nto find in the test config to modify the code. Since i modifed the existing code. Lets discuss on this
> * delete subsystem is deleting other subsystem(nqn.2016-06.io.spdk:cnode2) than the one specified in test-
--- Deletion is working fine- I had used different subsystems names for validation hence logs says cnod2 and test config says cnod1 -Whatever was created is deleted. Retained the subsystem name instead of the one which i tested.
> * on client, nvme disconnect is getting applied on other subsystem(nqn.2016-06.io.spdk:cnode2) than one specified in test -- Same as above
> * additionally nvme-cli is getting installed again if I am not wrong.
> ---------This is expected as per the disucssion with sunil, We are ensuring the nvme-cli and modprobe on the initaor node - Since its not harming or bothering the disconnect we decided to keep it
> Could you please check these if this is the case?